### ISSUE TITLE
question about test_view_interp.py

### ISSUE THREAD

**Issue by One-coder-gzh** [aipixel/GPS-Gaussian#68]
![image](https://github.com/user-attachments/assets/55672495-8d43-490b-bdea-5c134a664ed9)
do we need to replace data['novel_view']['img_pred'] with data_i['novel_view']['img_pred']

**Comment by ShunyuanZheng** [comment]
Yes, you are right. However, it is confusing that this bug seems to have no influence on the final result. The using of data and data_i generates the same results : )

**Comment by One-coder-gzh** [comment]
Thanks for your reply and your great work!
















At 2024-10-26 16:07:16, "Sawyer" ***@***.***> wrote:

Yes, you are right. However, it is confusing that this bug seems to have no influence on the final result. The using of data and data_i generates the same results : )
